microcosm_eventsource/stores/rollup.py

```diff
@@ -30,6 +42,7 @@ def container_type(self):
     def event_type(self):
         return self.event_store.model_class
 
+    @postgres_metric_timing(action="retrieve")
```

Comment:
I'm going to assume these are all reported as `extended_<model>`? Just checking, in case it was accidentally labeled with the non-extended name.

Reply:
right. postgres_metric_timing decorator uses model_name property.